### ISSUE TITLE
bolt12 regex is probably incomplete

### ISSUE THREAD

**Issue by daGoodenough** [daGoodenough/bolt12-prism#1]

**Comment by daGoodenough** [comment]
@Harshal662 are you familiar with bolt12 offers? This could be a good issue to tackle.

**Comment by Harshal662** [comment]
No, I have to read about it. As the time permits I can work on it. But, I am currently busy with my academic project. Hope someone with knowledge of bolt2 can do it.
